src/components/file-browser/NoticeForPath.ts

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash/fp'
+import { div } from 'react-hyperscript-helpers'
+import colors from 'src/libs/colors'
+
+
+type NoticeForPathProps = {
+  notices: { [path: string]: string }
+  path: string
+}
+
+export const NoticeForPath = (props: NoticeForPathProps) => {
+  const { notices, path } = props
```

Comment:
Could be

```suggestion
export const NoticeForPath = (({ notices, path }): NoticeForPathProps) => {
```

concision.

Reply:
I've started tending towards this style for TypeScript in order to separate the type definition from the props destructuring. In components with more props, having both in the same in the same place can get cluttered.

Reply:
Yea I wouldn't destructure in the definition if there are more than 2-3 props either.